homeassistant/components/unifi/device_tracker.py

```diff
@@ -244,17 +244,17 @@ def device_state_attributes(self):
     async def options_updated(self) -> None:
         """Config entry options are updated, remove entity if option is disabled."""
         if not self.controller.option_track_clients:
-            await self.async_remove()
+            await self.remove_item(self.client.mac)
```

Comment:
Is `self.client.mac` a set of addresses?

Reply:
... No >.<. Thank you